scripts/blocks-utils.js

```diff
@@ -60,8 +60,18 @@ const formatDateTime = (date) => date && date.toLocaleString('en-US', {
   hour12: true,
 });
 
+function addIcon(parentElement, iconName, prefix = '', alt = '') {
+  const img = document.createElement('img');
+  img.dataset.iconName = iconName;
+  img.src = `${window.hlx.codeBasePath}${prefix}/icons/${iconName}.svg`;
+  img.alt = alt;
+  img.loading = 'lazy';
```

Comment:
we can take `loading='lazy'` as param, since its util and someone might want to override it